src/components/Book.js

```diff
@@ -1,22 +1,36 @@
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bookRemoveAction } from '../actions';
 
 function Book(props) {
-  const { book } = props;
+  const { book, bookRemoveAction } = props;
+
+  const handleRemoveBook = (book) => {
+    bookRemoveAction(book);
+  };
```

Comment:
This method is expected to be implemented in the `BookList` component and passed down to the book as a prop. [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)